src/cache-redis/cache-redis.module.ts

```diff
@@ -0,0 +1,8 @@
+import { Module } from "@nestjs/common";
+import { CacheRedisService } from "./cache-redis.service";
+
+@Module({
+	providers: [CacheRedisService],
+	exports: [CacheRedisService,]
+})
+export class CacheRedisModule {}
```

Comment:
Don't name the module `CacheRedis` this doesn't use the cache at all, it calls an API that will get the conversion cache.

Reply:
Use another name like `DbioConversion` instead.